prime-angular-frontend/src/app/modules/adjudication/shared/components/search-form/search-form.component.ts

```diff
@@ -22,15 +23,22 @@ export class SearchFormComponent implements OnInit {
   public form: FormGroup;
   public statuses: Config<number>[];
 
+  private textSearchKey: string;
+  private statusCodeKey: string;
+
   constructor(
     private route: ActivatedRoute,
     private fb: FormBuilder,
     private configService: ConfigService,
+    private localStorage: LocalStorageService
   ) {
     this.statuses = this.configService.statuses;
     this.search = new EventEmitter<string>();
     this.filter = new EventEmitter<EnrolmentStatusEnum>();
     this.refresh = new EventEmitter<void>();
+
+    this.textSearchKey = 'search-form-textSearch';
+    this.statusCodeKey = 'search-form-statusCode';
```

Comment:
Since these two keys reside in a shared component, suspect it would cause problem when being used in multiple pages, and values could be overwritten/loaded accidentally.